AMDirT/cli.py

```diff
@@ -0,0 +1,259 @@
+@cli.command()
+@click.argument("samples", type=click.Path(exists=True))
+@click.argument("table_name", type=str)
+@click.option(
+    "-t",
+    "--tables",
+    type=click.Path(exists=True),
+    help="(Optional) JSON file listing AncientMetagenomeDir tables",
+)
+@click.option(
+    "-o",
+    "--output",
+    type=click.Path(writable=True, dir_okay=True, file_okay=False),
+    default=".",
+    show_default=True,
+    help="conversion output directory",
+)
+@click.option(
+    "--bibliography",
+    is_flag=True,
+    help="Generate BibTeX file of all publications in input table",
+)
+@click.option(
+    "--librarymetadata",
+    is_flag=True,
+    help="Generate AncientMetagenomeDir libraries table of all samples in input table",
+)
+@click.option(
+    "--curl",
+    is_flag=True,
+    help="Generate bash script with curl-based download commands for all libraries of samples in input table",
+)
+@click.option(
+    "--aspera",
+    is_flag=True,
+    help="Generate bash script with Aspera-based download commands for all libraries of samples in input table",
+)
+@click.option(
+    "--eager",
+    is_flag=True,
+    help="Convert filtered samples and libraries tables to eager input tables",
+)
+@click.option(
+    "--fetchngs",
+    is_flag=True,
+    help="Convert filtered samples and libraries tables to nf-core/fetchngs input tables",
+)
+@click.option(
+    "--ameta",
+    is_flag=True,
+    help="Convert filtered samples and libraries tables to aMeta input tables",
+)
+@click.option(
+    "--mag",
+    is_flag=True,
+    help="Convert filtered samples and libraries tables to nf-core/mag input tables",
+)
+@click.option(
+    "--taxprofiler",
+    is_flag=True,
+    help="Convert filtered samples and libraries tables to nf-core/taxprofiler input tables",
+)
+@click.pass_context
+def convert(ctx, no_args_is_help=True, **kwargs):
+    """\b
+    Converts filtered samples and libraries tables to eager, ameta, taxprofiler, and fetchNGS input tables
+    \b
+    SAMPLES: path to filtered ancientMetagenomeDir samples tsv file
+    TABLE_NAME: name of table to convert
+    """
+    run_convert(**kwargs, **ctx.obj)
+
+
+#################
+# Autofill tool #
+#################
+
+@cli.command()
+@click.argument("accession", type=str, nargs=-1)
+@click.option(
+    "-n",
+    "--table_name", 
+    type=click.Choice(get_table_list()),
+    default='ancientmetagenome-hostassociated',
+    show_default=True
+)
+@click.option(
+    "-l",
+    "--library_output",
+    type=click.Path(writable=True),
+    help="path to library output table file"
+)
+@click.option(
+    "-s",
+    "--sample_output",
+    type=click.Path(writable=True),
+    help="path to sample output table file"
+)
+@click.pass_context
+def autofill(ctx, no_args_is_help=True, **kwargs):
+    """\b
+    Autofills library and/or sample table(s) using ENA API and accession numbers
+    \b
+
+    ACCESSION: ENA accession(s). Multiple accessions can be space separated (e.g. PRJNA123 PRJNA456)
```

Comment:
I would replace the multiple accession examples to sample accessions rather than PRJ, as this is going to be a more natural use by a user (maybe even have two exampes: single: PRJ or multiple: ERS ERS)